<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddIngredienten</class>
 <widget class="QDialog" name="AddIngredienten">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Afsluiten zonder opslaan</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/stop2.png</normaloff>:/icons/icons/stop2.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNew">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Nieuw</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Opslaan</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/save_close.png</normaloff>:/icons/icons/save_close.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wissen</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/cross.png</normaloff>:/icons/icons/cross.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Aanpassen</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/edit_button.png</normaloff>:/icons/icons/edit_button.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tvIngredienten">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>135</y>
     <width>541</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbIngredient">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingredient:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnToevoegen">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>15</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Nieuw</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxIngredienten">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>15</y>
      <width>356</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Kies hier uw toe te voegen ingredient</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbHoeveelheid">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>176</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hoeveelheid (in procent):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbHoeveelheid">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>lbHoeveelheid</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
